FastBank.Domain/Message.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace FastBank.Domain
+{
+    public class Message
+    {
+        public Message(Guid messageId,
+                       Customer sender,
+                       Customer? receiver,
+                       Roles receiverRole,
+                       string text,
+                       string subject,
+                       Message? basedOnMessage,
+                       MessageStatuses messageStatuses,
+                       MessageType messageType)
```

Comment:
![image](https://github.com/bulgarchik/FastBank/assets/108490889/0e134cec-5abc-42ff-b3c8-e19db62e357d)
See my example of correctly wrapped method parameters